src/Cli/dotnet/commands/dotnet-test/TestingPlatformCommand.cs

```diff
@@ -41,13 +69,16 @@ public int Run(ParseResult parseResult)
             ForwardingAppImplementation msBuildForwardingApp = new(
                 GetMSBuildExePath(),
                 [$"-t:{(containsNoBuild ? string.Empty : "Build;")}_GetTestsProject",
+                "-bl",
                         $"-p:GetTestsProjectPipeName={_pipeNameDescription.Name}",
                         "-verbosity:q"]);
-            int getTestsProjectResult = msBuildForwardingApp.Execute();
+            _ = msBuildForwardingApp.Execute();
```

Comment:
I think we should verify that the exit code is 0 or something went wrong inside the msbuild task...can you try to simulate a bug(throw Exception() inside the get task)?

If we failed here we should "print something and exit"